src/bin/bors.rs

```diff
@@ -1,34 +1,65 @@
 use std::net::SocketAddr;
+use std::path::PathBuf;
 use std::sync::Arc;
 
 use anyhow::Context;
+use axum::extract::State;
 use axum::http::StatusCode;
 use axum::response::IntoResponse;
 use axum::routing::post;
 use axum::Router;
 use clap::Parser;
 use tower::limit::ConcurrencyLimitLayer;
 
-use bors::config::Config;
+use bors::github::api::RepositoryAccess;
+use bors::github::service::{github_webhook_process, WebhookSender};
 use bors::github::webhook::GitHubWebhook;
+use bors::github::WebhookSecret;
 
 #[derive(clap::Parser)]
 struct Opts {
     /// Secret used to authenticate webhooks.
     #[arg(long, env = "WEBHOOK_SECRET")]
     webhook_secret: String,
+    /// Github App ID.
+    #[arg(long, env = "APP_ID")]
+    app_id: u64,
+    /// Path to a private key used to authenticate as a Github App.
+    #[arg(long, env = "PRIVATE_KEY_PATH")]
+    private_key_path: PathBuf,
```

Comment:
I think in production this probably will be an env variable, not a path to some file.

Reply:
Yeah, the private key content will be in the environment variable.